### PR TITLE
Create admin.py

### PR DESCRIPTION
**Pull request by muhammedfurkan**

### CODE REVIEW

File: pyrobot/plugins/admin.py [hunk 1]
```diff
@@ -0,0 +1,435 @@
+
+import asyncio
+import logging
+import math
+import os
+from asyncio import sleep
+from datetime import datetime
+from os import remove
+from time import sleep, time
```
Comment (SpEcHiDe) [inline pyrobot/plugins/admin.py:9]: don't do this...

Comment (muhammedfurkan) [reply pyrobot/plugins/admin.py:9]: why

File: pyrobot/plugins/admin.py [hunk 2]
```diff
@@ -0,0 +1,435 @@
...
+        await asyncio.sleep(2)
+        await message.delete()
+    elif message.reply_to_message.from_user.is_self:
+        await message.edit(f"I can't {message.command[0]} myself.")
+        await asyncio.sleep(2)
+        await message.delete()
+    else:
+        return True
+
+async def AdminCheck(message):
```
Comment (SpEcHiDe) [inline pyrobot/plugins/admin.py:94]: import the AdminCheck from helper_funcs, avoids duplicate code.

File: pyrobot/plugins/admin.py [hunk 3]
```diff
@@ -0,0 +1,435 @@
...
+        elif SELF.permissions.can_restrict_members:
+            return True
+        else:
+            await message.edit("__No permissions to restrict Members__")
+
+async def RestrictFailed(message):
+    await message.edit(f"I can't {message.command[0]} this user.")
+
+
+@Client.on_message(Filters.command(r"setgic", ".")  & Filters.me)
```
Comment (SpEcHiDe) [inline pyrobot/plugins/admin.py:116]: use the command handler variable.

Comment (muhammedfurkan) [reply pyrobot/plugins/admin.py]: done

File: pyrobot/plugins/admin.py (hunk 1 above)
Comment (SpEcHiDe) [reply pyrobot/plugins/admin.py:9]: also that import is not used anywhere, right? Try removing the unused imports, 

and try making different commands into a seperate file. 

Maybe, admin, directory and inside that different files. 🤔😬😬

Comment (muhammedfurkan) [reply pyrobot/plugins/admin.py:9]: okay i will do it. thanks.

**Pull request closed without merging**